src/domain/Modals/Modal/style.tsx

```diff
@@ -170,6 +170,17 @@ const StyledReactModal = styled(ReactModalAdapter)`
         width: 100%;
       }
     }
+
+    &.modal-size-fullscreen {
+      min-width: 0;
+      width: auto;
+
+      @media only screen and (min-width: ${breakpointTablet}px) {
+        min-height: 100vh;
```

Comment:
There is a 1rem padding on the top and bottom around the modal. 
So I think you want this to be `calc(100vh - 2rem)` so it fits nice on the screen if there isn't enough content to force a scroll.